### PR TITLE
Preprocess tod updates

### PR DESCRIPTION
**Pull request by davidvng**
- Added `HWPAngleModel` to preprocess
- Added `select` to `SourceFlags`
- Added optional `rn_range` and `si_nan` parameters to `get_det_bias_flags`
- Added verbosity to `preprocess_tod` and modified `util.init_logger` correspondingly to allow updating handler verbosity

### CODE REVIEW

File: sotodlib/hwp/hwp_angle_model.py [hunk 1]
```diff
@@ -34,14 +36,14 @@ def apply_hwp_angle_model(tod, on_sign_ambiguous='fail'):
 
     methods = ['pid', 'offcenter']
     if on_sign_ambiguous in methods:
-        sign = hwp[on_sign_ambiguous + '_direction']
+        sign = hwp.get(on_sign_ambiguous + '_direction', 0)
```
[the remark this answers is not in the document]
Comment (davidvng) [reply sotodlib/hwp/hwp_angle_model.py:39]: I had a case where there was no "offcenter_direction" key and so it gave a `KeyError` and crashed, I think there was only "offcenter"

[the remark this answers is not in the document]
Comment (msilvafe) [reply sotodlib/hwp/hwp_angle_model.py:39]: Should this add a check for the version number then?

[the remark this answers is not in the document]
Comment (ykyohei) [reply sotodlib/hwp/hwp_angle_model.py:39]: I don't think we add version check. I made breaking change from 240430m for improvement of hwp_angle correction, but I will not make change which requires version check in future. Also, people should use latest version.

**Review by msilvafe — APPROVED** [review]
Thanks for working with @ykyohei on the hwp_angle_model it will be important update until we get a new loader. Good to get the det flags in alignment with the satp3 planet folks workflow as well.

**Pull request merged**